SageOfCode/Crate
# User association style

# What does this PR do?

 * Add the styleByUser Query
 * Add userQuery tests

# Where should the reviewer start?

* Drop and create Crate and Crate_Test

# How should this be manually tested?

* Pull down branch and run "npm start" in code/web and code/api to ensure that the app is running
* Ensure you have the dev tools and look into the Redux section

# Any background context you want to provide?

* This allows the userId can be used to query the Style table and return the description and image_url
* This allows the query for an id that finds the description and image_url for the Style table
What are the relevant tickets?
#47 
#66 
#62
#43 
#65
